README.md

```diff
@@ -1,56 +1,74 @@
-# ember-sbe-tooling [![Circle CI][circleci-status]][circleci]
+# ember-sbe-tooling
 
-This addon is for internal use on Showbie's Ember projects to enforce a consistent coding standard. We will not accecpt external pull requests but feel free to take inspiration from it.
+[![npm version][npm-ver]][npm]
+[![Circle CI][circleci-status]][circleci]
+
+This addon is for internal use on Showbie's Ember projects to enforce a 
+consistent coding standard. We will not accept external pull requests but feel 
+free to take inspiration from it.
 
 ## Usage
 
-```
+```sh
 ember install ember-sbe-tooling
 ```
 
-To update the addon, you should be able to simply re-run the above command. If needed you can also run the blueprints independently with `ember g showbie-tooling`.
-
-Note that you should ensure you are using the appropriate version of Node:
+To update the addon, you should be able to simply re-run the above command. If 
+needed you can also run the blueprints independently:
 
+```sh
+ember g showbie-tooling
 ```
+
+Note that you should ensure you are using the appropriate version of Node/npm:
+
+```sh
 nvm use
 ```
 
 ## Style enforcement
 
 This addon provides configuration for the following tools:
 
-- nvm
+- CircleCI
 - editorconfig
-- testem
+- ember-cli-release
 - eslint
-- scss-lint
+- nvm
 - pre-commit
-- CircleCI
-- ember-cli-release
+- prettier
+- stylelint
+- testem
+- yarn
 
 ## IDE integration
-For realtime feedback in your editor, you'll likely need to install [eslint-plugin-ember-suave][ember-suave-plugin] globally:
 
+For realtime feedback in your editor, you may need to install 
+[eslint-plugin-ember][] globally:
```

Comment:
I think you're missing the actual link to `eslint-plugin-ember` here.

Reply:
It's a reference-style link, it's at the bottom of the file :)

Reply:
I know it's a reference link—I just didn't think you could leave the second brackets empty. Does that work!?

Reply:
Yep! http://www.cirosantilli.com/markdown-style-guide/#reference-style-links

You can technically leave off the empty `[]` afterward and most Markdown parsers will figure it out.